e2e/protractor.conf.js

```diff
@@ -15,7 +15,7 @@ exports.config = {
   jasmineNodeOpts: {
     showColors: true,
     defaultTimeoutInterval: 30000,
-    print: function() {},
+    print: function () {},
```

Comment:
Could you please amend your comit and run `yarn run prettier:write` ?

There's a bit of noise in that PR due to a different formatting from your IDE I believe

Reply:
Oh NVM I just noticed you did update Prettier in package.json, I assume that's what triggered those changes

Reply:
Yea, that's actually the result of `yarn run prettier:write`, I don't like it as well. Probably it would make sense to make a config update separately.